client/src/components/FoodSeeker/SearchResults/layouts/Desktop.js

```diff
@@ -87,7 +87,7 @@ const DesktopLayout = ({ filters, list, map }) => {
             left: isListPanelOpen ? 0 : "-524px",
             top: "120px",
             height: "100%",
-            zIndex: "1",
+            zIndex: "3",
             background: "white",
           }}
         >
```

Comment:
remove z index from the button on line 116. The shadow comes from the svg file.

Reply:
This ticket is for mapbox logo's z-index. I think you are mistaken #2084 with #2080

Reply:
Nit: For consistency and best practice, if a z-index value of 3 achieves the desired effect, it's recommended to use the numeric value (3) instead of the string ('3').

Reply:
Considering this pull request addresses issue #2084, I'll proceed with opening a new pull request for issue #2080. Please ignore my previous suggestion about removing lines from the `DrawerLeftArrowButton.js` file.

Reply:
fixed